unix/Makefile

```diff
@@ -156,11 +156,13 @@ LIB_SRC_C = $(addprefix lib/,\
 	timeutils/timeutils.c \
 	)
 
+ifeq ($(MICROPY_VFS_FAT), 1)
```

Comment:
Are you sure this ifeq is needed?  The VFS stuff is disabled by default (eg not included in standard unix build).